common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -121,6 +121,19 @@ private static async Task Run(Type testType, PerfOptions options)
                             await RunTestsAsync(tests, options, title);
                         }
                     }
+                    catch (AggregateException ae)
+                    {
+                        foreach (Exception e in ae.InnerExceptions)
+                        {
+                            Console.WriteLine($"Exception: '{e.Message}'{Environment.NewLine}{e.StackTrace}");
```

Comment:
```suggestion
                            Console.WriteLine($"Exception: {e}");
```

I think this code is sufficient, since `Exception.ToString()` includes the message and stack trace by default.